src/main/java/seedu/address/model/student/Attendance.java

```diff
@@ -11,7 +11,7 @@ public class Attendance {
 
     public static final String MESSAGE_CONSTRAINTS =
             "Attendance should only contain numbers";
```

Comment:
Can also change this to "only contain nonnegative whole numbers"

Reply:
Done